.pre-commit-config.yaml

```diff
@@ -0,0 +1,51 @@
+repos:
+- repo: https://github.com/psf/black
+  rev: 21.12b0
```

Comment:
I use black 22.1.0
https://github.com/pypa/wheel/blob/main/.pre-commit-config.yaml use the same version and the option without "" and without space.

Reply:
Thanks @marcelotduarte. I've been out of this PR for a few days, but I think today I will be experimenting locally to fix the issue. Thanks for the reminder! :)

Reply:
ok, I'll fix it.